lib/vapi/ConnectionDevices.py

```diff
@@ -0,0 +1,95 @@
+# IBM_PROLOG_BEGIN_TAG
+#
+# Copyright 2020 IBM International Business Machines Corp.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#           http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+#  implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  IBM_PROLOG_END_TAG
+
+
+import logging as logger
+import os
+import requests
+import json
+
+
+class ConnectionDevices:
+
+    def __init__(self, server):
+        self.server = server
+
+    def create(self, jsonRequest):
+        """ Create a new Connection script.
+
+        :param name   -- name for the new connection script
```

Comment:
There is no `param` called name, just jsonRequest.
I would like to see `jsonRequest` named for the purpose it serves, not what it is (e.g. `connectionInfo`, or something similar).
This applies to `update()` also.

Reply:
Changed